src/core/prototypes/links/utils.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { ChartClass } from "../charts";
+import { LinksProperties } from "./index";
+import { PolarPlotSegment } from "../plot_segments";
+
+export function shouldShowCloseLink(
+  parent: ChartClass,
+  linkProperties: LinksProperties,
+  close?: boolean
+): boolean {
+  if (linkProperties.linkTable) {
+    return false;
+  }
+
+  if (!parent) {
+    return false;
+  }
+
+  if (!parent.object.elements) {
+    return false;
+  }
+
+  if (linkProperties.linkThrough) {
+    const plotSegment = parent.object.elements.find(
```

Comment:
Is it possible for the parent chart element to have multiple elements that satisfy this condition? Here `find` will return the first element that satisfies the linkThrough condition. For example, multiple Polar plot lines on the same chart?

Reply:
In this case, no. `linkThrough.plotSegment` has an `_id` on `plotSegment`. And in chart we have only one `plotSegment` with this `_id `